### PR TITLE
Update uninstall

### PR DESCRIPTION
**Pull request by amy-hashi**
Uninstall script will now check the default path for snapshots and prompt the user for action instead of just deleting them.

## Labels

- [ ] inaccuracy
- [x] clarification
- [ ] new docs
- [ ] cosmetic bug - fixing broken text or markup
- [ ] enhancement - changing the website's behavior/layout
- [ ] api - requires an update to the [changelog](https://www.terraform.io/docs/cloud/api/changelog.html)

### CODE REVIEW

File: content/source/docs/enterprise/install/uninstall.html.md [hunk 1]
```diff
@@ -20,10 +20,28 @@ After you initiate `uninstall`:
...
+
+	If there are snapshots present on the system, you will be prompted to move them, proceed with deleting them, or abort.
+	
+	```
+	There appear to be Replicated snapshots stored in /var/lib/replicated/snapshots.
+	1) Move the snapshots to another directory
+	2) Continue uninstall and delete the snapshots
+	3) Cancel the uninstall
+	Select an option: 1
+	Enter the directory to move the snapshots to:/tmp
```
Comment (sudomateo) [inline content/source/docs/enterprise/install/uninstall.html.md:37]:
```suggestion
	Enter the directory to move the snapshots to: /tmp
```

File: content/source/docs/enterprise/install/uninstall.html.md [hunk 2]
```diff
@@ -20,10 +20,28 @@ After you initiate `uninstall`:
...
+	The snapshots will be moved to /tmp.
+	Press Y to continue or N to cancel.yMoving snapshots...
```
Comment (sudomateo) [inline content/source/docs/enterprise/install/uninstall.html.md:39]:
```suggestion
	Press Y to continue or N to cancel. y
	Moving snapshots...
```

File: content/source/docs/enterprise/install/uninstall.html.md [hunk 3]
```diff
@@ -20,10 +20,28 @@ After you initiate `uninstall`:
 1. Enter `yes` when asked if you want to continue with the installation.
 
 	```
-	$ sudo ./uninstall.sh
-	This script will completely uninstall Terraform Enterprise and Replicated on this system, as well as remove associated files. This includes the Replicated snapshot directory. Please back up /var/lib/replicated/snapshots if you wish to save them.
+	$ sudo ./uninstall.sh
```
Comment (sudomateo) [inline content/source/docs/enterprise/install/uninstall.html.md:23]:
```suggestion
	$ sudo ./uninstall.sh
```

**Review by sudomateo — COMMENTED** [review]
Minor formatting changes.

File: content/source/docs/enterprise/install/uninstall.html.md (hunk 1 above)
Comment (amy-hashi) [reply content/source/docs/enterprise/install/uninstall.html.md:37]: I knew someone was gonna comment on that. I gotta update the script to fix this. (This is all copy-pasta directly from actual script output.

File: content/source/docs/enterprise/install/uninstall.html.md [hunk 4]
```diff
@@ -21,9 +21,28 @@ After you initiate `uninstall`:
 
 	```
 	$ sudo ./uninstall.sh
-	This script will completely uninstall Terraform Enterprise and Replicated on this system, as well as remove associated files. This includes the Replicated snapshot directory. Please back up /var/lib/replicated/snapshots if you wish to save them.
+	This script will completely uninstall Terraform Enterprise and Replicated on this system, as well as remove associated files.
 	Do you wish to continue? (y/n)yes
 	Proceeding with uninstall...
+	```
+
+	If there are snapshots present on the system, you will be prompted to move them, proceed with deleting them, or abort.
```
Comment (laurapacilio) [inline content/source/docs/enterprise/install/uninstall.html.md:29]:
```suggestion
	If there are snapshots present on the system, choose whether to move them to another directory, delete them, or cancel the uninstall. Here is an example of moving snapshots to another directory:
```

**Review by sudomateo — APPROVED** [review]

File: content/source/docs/enterprise/install/uninstall.html.md [hunk 5]
```diff
@@ -21,9 +21,28 @@ After you initiate `uninstall`:
...
+	2) Continue uninstall and delete the snapshots
+	3) Cancel the uninstall
+	Select an option: 1
+	Enter the directory to move the snapshots to: /tmp
+	The snapshots will be moved to /tmp.
+	Press Y to continue or N to cancel. y
+	Moving snapshots...
+	Files moved.
+	```
+	The uninstall will then continue...
```
Comment (laurapacilio) [inline content/source/docs/enterprise/install/uninstall.html.md:43]: I would do this here, and remove that line from below the example block. I think this is clearer. 

```suggestion
	The script continues and stops the Replicated services, removes the Docker containers, and removes Replicated executables and configuration files from the system.  
```

**Review by laurapacilio — APPROVED** [review]
Two suggestions for you - thank you!

**Pull request merged**